### ISSUE TITLE
Authentication doesn't seem to work

### ISSUE THREAD

**Issue by dbwelch** [xamarinhq/app-evolve#56]
The authentication used to work with Xamarin.com ids, but, now it does not recognize this id or a current Microsoft.com ID.  Is the authentication path still functional?

**Comment by jamesmontemagno** [comment]
Yes, we stopped using Xamarin Authentication a while ago. I need to take it out completely and mock out a login.
